experiment/src/prepare_data.py

```diff
@@ -0,0 +1,34 @@
+import os
+import sys
+import subprocess
+
+
+def execute_scanner(dataset_location: str, result_location_str, j, use_ml=False):
+    """Execute CredSweeper as a separate process to make sure no global states is shared with training script"""
+    dir_path = os.path.dirname(os.path.realpath(__file__)) + "/.."
+    command = f"{sys.executable} -m credsweeper --path {dataset_location}/data --save-json {result_location_str} -j {j}"
+    if use_ml:
+        command += " --ml_validation"
+    subprocess.call(command, shell=True, cwd=dir_path, stdout=subprocess.DEVNULL, stderr=subprocess.STDOUT)
```

Comment:
Why not to use credsweeper as a library?

Reply:
Oh, it's to guaranty total independence of memery
CredSweaper still have `MlValidator` as a global (static) object, and we use TF/Keras. TF/Keras also like to spam global objects. So I'm scared that it will somehow influence the training results